core/messagehelper.go

```diff
@@ -15,7 +15,7 @@ func GetAppliedContentFunction(applyTo string) GetAppliedContent {
 
 	if parts[0] == "meta" {
```

Comment:
TBD
The "meta" keyword does not add any information to the argument. We only use it so that a metadata key called "payload" is allowed, too. Without that we could just say "anything that is NOT payload is treated as a metadata key".
... if you continue thinking this way, "payload" is just a special metadata field ... interesting.
That makes the payload field an access-time optimisation.